### PR TITLE
Fixes to the sample

### PR DESCRIPTION
**Pull request by NoahAndrews**
* Makes it possible for `npx rev-hub-cli` to work once we've published it
* Adds missing dependency on Typescript
* Fixes broken scripts in `package.json`

### CODE REVIEW

File: packages/sample/package.json [hunk 1]
```diff
@@ -3,20 +3,23 @@
...
+        "typescript": "^5.0.2"
+    },
     "scripts": {
         "build": "tsc",
         "motor": "node dist/main.js motor power 1 0.60",
         "analog": "node dist/main.js analog 0",
-        "battery": "node dist/main.js battery --continuous",
-        "voltage": "node dist/main.js voltage --continuous",
+        "voltage": "node dist/main.js battery voltage --continuous",
+        "current": "node dist/main.js battery current --continuous",
```
Comment (LandryNorris) [inline packages/sample/package.json:22]: Do we still want to keep these scripts? We don't have full coverage of the sample via these scripts, so I would be ok with removing them.

File: packages/sample/src/main.ts [hunk 2]
```diff
@@ -715,7 +720,7 @@ program
         await runServo(hub, channelValue, pulseWidthValue, frameWidthValue);
     });
 
-program.parse(process.argv);
+await program.parseAsync(process.argv);
```
Comment (LandryNorris) [inline packages/sample/src/main.ts:723]: I'm unclear on why this is here. Is parse not blocking?

File: packages/sample/package.json (hunk 1 above)
Comment (NoahAndrews) [reply packages/sample/package.json:22]: Fair point, I'll just do that

File: packages/sample/src/main.ts (hunk 2 above)
Comment (NoahAndrews) [reply packages/sample/src/main.ts:723]: I actually didn't notice I was committing that, I'll get rid of it. This whole PR originated from my WebStorm Shelf.

**Review by LandryNorris — APPROVED** [review]

**Pull request merged**